app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"All {visitor['name']} should be vaccinated")
```

Comment:
The error message in `NotVaccinatedError` should be corrected to: `f"{visitor['name']} should be vaccinated"` to match the task description and provide a more accurate message.